### ISSUE TITLE
WCAG - Update the Video transcript link

### ISSUE THREAD

**Issue by fionamorrison23** [civictheme/uikit#759]
### User story

As an anonymous user
I want view the transcript on the same page
So that the transcript does not open a new page in the same window therefore navigating me away from the video

### Description

Relates to: https://github.com/civictheme/uikit/issues/706 
We want to update the video transcript link so the transcript is a collapsible element instead of linking to a new page. This means we should also update the element used to trigger the transcript from a link to a button.
Best practice says that links should be used to navigate and buttons should be used for an action.
Hence the video transcript link should be updated to use a button.

### Acceptance criteria

**AC 1 - Collapsible**
Given I am an anonymous user
When I click on the "View video transcript" element
Then a collapsible element opens
And the video transcript is displayed inside the collapsible element

**AC 2 - Button**
Given I am an anonymous user
When I inspect the "View video transcript" element
Then the element role is "button"
And it is of the button type "Tertiary"
And it is operable with both mouse and keyboard input

**AC 3 - Aria labels*
Given I am an anonymous user
When the "View video transcript" element is rendered
Then it has aria-expanded set to "false" by default
And when the transcript is expanded
Then aria-expanded is set to "true"
And the collapsible region has an aria-controls attribute linking to the transcript container

**AC 2 - Screen reader**
Given I am using a screen reader
When I navigate to the "View video transcript" button
Then the screen reader should announce it as a button
And should indicate whether the transcript is expanded or collapsed
